python-sdk/tests/benchmark/dags/evaluate_load_file.py

```diff
@@ -1,21 +1,26 @@
 import json
 import os
 from datetime import datetime
-from pathlib import Path
 
 from airflow import DAG
 from astro import sql as aql
 from astro.constants import DEFAULT_CHUNK_SIZE, FileType
 from astro.files import File
 from astro.sql.table import Metadata, Table
+from google.cloud import storage
+from tests.benchmark import settings as benchmark_settings
 
 START_DATE = datetime(2000, 1, 1)
 
 
 def load_config():
-    config_path = Path(Path(__file__).parent.parent, "config.json").resolve()
-    with open(config_path) as fp:
-        return json.load(fp)
+    tmp_config = "/tmp/config.json"
+    storage_client = storage.Client()
+    storage_client.download_blob_to_file(
```

Comment:
It may be worth having it in a versioned GCS bucket and printing the object's version.
I think it is important that we print the content of this file for troubleshooting.